test/Intl/DateTimeFormatOptions.js

```diff
@@ -51,7 +51,7 @@ var tests = [
                 }
                 catch (e) {
                     if (!(e instanceof RangeError)) {
-                        fail("Incorrect exception was thrown.");
+                        assert.fail("Incorrect exception was thrown.");
```

Comment:
I don't remember exactly. It was a bug linked to my changes and it was failing on this test, but not only. I fixed the problem and everything was working fine again. There was no direct link with this file but I considered that it was good to fix this line by the way.